### ISSUE TITLE
memoryerror during static string extraction

### ISSUE THREAD

**Issue by williballenthin** [fireeye/flare-floss#165]
sample `e81f9dad...`:

```
Traceback (most recent call last):
  File "<string>", line 409, in <module>
  File "<string>", line 396, in main
  File "<string>", line 336, in print_stack_strings
  File "floss\stackstrings.py", line 132, in extract_stackstrings
  File "floss\strings.py", line 30, in extract_ascii_strings
MemoryError
main returned -1
```

**Comment by williballenthin** [comment]
works on some systems, not other. perhaps its memory exhaustion?

**Comment by williballenthin** [comment]
cc @0xF2EDCA5A

**Comment by williballenthin** [comment]
no longer exhibits crashes (nor uses more than 300MB of memory). perhaps related to fixes #228 #229 or #230
